index.html

```diff
@@ -4,9 +4,7 @@
 <head>
     <meta charset='UTF-8'>
     <title>JavaScript Example</title>
-    <script src="node_modules/moment/min/moment.min.js"></script>
-    <script src="moment.js"></script>
-    <script src='index.js'></script>
+    <script src="dist/main.js"></script>
```

Comment:
it was really not ok before.
```suggestion
    <script src="dist/main.js"></script>
```